Makefile

```diff
@@ -75,6 +77,10 @@ logs: ## display app logs (follow mode)
 .PHONY: logs
 
 run: ## start the wsgi (production) or development server
+	@$(COMPOSE) up -d redis-primary
+	@$(WAIT_REDIS)
+	@$(COMPOSE) up -d redis-sentinel
```

Comment:
Maybe it would make sense that the redis-sentinel service depend on the redis-primary service in docker-compose?

Reply:
The dependency is already defined in the docker-compose.
This was to make sure that `redis-primary` is reachable when `redis-sentinel` starts.